kolibri/plugins/coach/assets/src/vue/reports/summary-section/index.vue

```diff
@@ -1,255 +0,0 @@
-<template>
```

Comment:
This will be useful for boilerplating, since this is where the breadcrumb info is passed